### ISSUE TITLE
Change pattern on removing domain managers from tooltip to alert

### ISSUE THREAD

**Issue by gabydisarli** [cisagov/manage.get.gov#2711]
### Issue description

We're trying to avoid using tooltips when we can for accessibility reasons. In the Org Model designs, we're using a different pattern for messaging shown to users when a domain has only 1 domain manager on it. Currently, we disable the Remove button and show a tooltip on hover that tells the user they can't remove themselves because they are the sole domain manager. We'd like to change this to making the button clickable, but showing a top of page error alert that tells them they can't be removed.

**Note: this is a specific use case that applies to the button only when the domain has 1 domain manager on it.**

See [Figma design](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=8847-180114&node-type=FRAME&t=wUCdysiDcFKG8uGh-11) for reference

### Acceptance criteria

- [x] Change "Remove" button to the default state instead of disabled state
- [x] When user clicks the button, show top of page error alert.

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**

**Comment by gabydisarli** [comment]
cc: @AnnaGingle here's the issue to change the tooltip to the alert pattern we're using in Org model
